### PR TITLE
Fixed issue #16536: Batch editing reminder sent date leads to wrong data being added

### PR DESCRIPTION
**Pull request by gabrieljenik**
Reviewed fields initialization and jquery handling.
- Fixed some element IDs on the mass edit modal dialog.
- Changed update method of hidden input from attr('value',xxx) to val(xxx)
- Added initialization of YesNo and YesNoDate fields on "shown.bs.modal" and "actions-updated" event.

### CODE REVIEW

**Comment by sushmanadendlals** [comment]
Working as expected, Please refer Mantis for more details

File: assets/scripts/admin/tokens.js [hunk 1]
```diff
@@ -38,19 +38,19 @@ $.fn.YesNoDate = function(options)
             {
                 // Show date
                 $elDateContainer.show();
-                $elHiddenInput.attr('value', moment().format($elDate.data('date-format')));
+                $elHiddenInput.val(moment().format($elDate.data('date-format')));
```
Comment (olleharstedt) [inline assets/scripts/admin/tokens.js:41]: What's the difference?

Comment (gabrieljenik) [reply assets/scripts/admin/tokens.js:41]: When using val() the value is set in the DOM. When using attr, sometimes, that's not updated on the DOM

File: application/views/admin/token/massive_actions/_update.php [hunk 2]
```diff
@@ -424,7 +424,7 @@ class="YesNoDatePicker form-control action_datepickerUpdateHiddenField"
     formGroup.find('.YesNoSwitch').on('switchChange.bootstrapSwitch', function(e, state){
         
         formGroup.find('.selector_datechange').css('display', (state ? '' : 'none'));
-        formGroup.find('.selector_submitField').val(state ? 'Y' : 'N');
+        //formGroup.find('.selector_submitField').val(state ? 'Y' : 'N');
```
[the remark this answers is not in the document]
Comment (gabrieljenik) [reply application/views/admin/token/massive_actions/_update.php]: Fixed

File: application/views/admin/token/massive_actions/_update.php [hunk 3]
```diff
@@ -419,15 +419,12 @@ class="YesNoDatePicker form-control action_datepickerUpdateHiddenField"
...
-       console.log(\"bindBSSwitch run on:\",formGroup);
-    //Script to update the completed settings
-    formGroup.find('.YesNoSwitch').on('switchChange.bootstrapSwitch', function(e, state){
-        
-        formGroup.find('.selector_datechange').css('display', (state ? '' : 'none'));
-        //formGroup.find('.selector_submitField').val(state ? 'Y' : 'N');
-
-    });
-   };
+        console.log(\"bindBSSwitch run on:\",formGroup);
```
Comment (olleharstedt) [inline application/views/admin/token/massive_actions/_update.php:422]: Do we need this debugging statement?

Comment (olleharstedt) [reply application/views/admin/token/massive_actions/_update.php:422]: I don't remember if we had a proper logger in JS? That's only on if debug = 2, with different logging levels.

Comment (gabrieljenik) [reply application/views/admin/token/massive_actions/_update.php]: Fixed

**Pull request merged**